Tooltip Atom: reusable component

FariaAfrose opened the pull request:
### Description

Please describe your changes, better with bullet points.

### Preview

Share a preview image for better understanding.

### Task link

Share the JIRA task link.

### Review requests

Mention the reviewers

FariaAfrose committed:
chore: add untitledui/icons package

FariaAfrose committed:
i18n: add tooltip translations in English and Swedish

FariaAfrose committed:
feat: add tooltip to components drawer menu items

FariaAfrose committed:
chore: add COLOR exports in styles config

FariaAfrose committed:
feat(tooltip): add reusable Tooltip atom with subtitle and icon support

foysal-strativ-ab commented on src/components/Atoms/Tooltip/index.tsx:
```diff
@@ -0,0 +1,135 @@
+
+export type TooltipType = 'default' | 'with-subtitle';
+
+export type TooltipAtomProps = {
```
It is not necessary to add Atom in the naming apu. please remove the Atom part.

foysal-strativ-ab commented on src/config/styles.ts:
```diff
@@ -1,4 +1,4 @@
-const COLORS = {
+export const COLORS = {
```
we will not be exporting this object, Our goal is to make the colors accessible only through the theme.

foysal-strativ-ab commented on src/components/Atoms/Tooltip/index.tsx:
```diff
@@ -0,0 +1,135 @@
+  title: ReactNode;
+  subtitle?: ReactNode;
+  size?: TextVariant;
+  weight?: WeightVariant;
```
Apu why are we controlling the weight, lineHeight, and padding of the texts?
I think we are supposed to match the Figma only. No need to make those flexible. If for some unique project they need a different fonts, it can be cutomized then according to the project requirements.

foysal-strativ-ab commented on src/components/Atoms/Tooltip/index.tsx:
```diff
@@ -0,0 +1,135 @@
+  subtitle?: ReactNode;
+  size?: TextVariant;
+  weight?: WeightVariant;
+  lineHeight?: LineHeightVariant;
```
Let's omit weight, padding, lineHeight, and borderRadius from the props, and use the Figma design values.

foysal-strativ-ab commented on src/components/Atoms/Tooltip/index.tsx:
```diff
@@ -0,0 +1,135 @@
+        ? { pointAtCenter: true }
+        : true;
+
+  const tooltipTrigger = children || helperIcon || (
```
Please style attribute as less as possible. Better make a Styled component and use as component.
Our convention to create styled component is: Create the component at the bottom of the file and reuse in the component

FariaAfrose committed:
refactor(ui): enforce Figma text styles with styled components

- Removed props for weight, lineHeight, padding, and borderRadius
- Minimized inline styles by using styled components

foysal-strativ-ab reviewed (approved):
Good work
